engines/order_management/app/services/order_management/stock/package.rb

```diff
@@ -97,6 +97,10 @@ def shipping_methods
         available_shipping_methods.keep_if do |shipping_method|
           ships_with?(order.distributor.shipping_methods.to_a, shipping_method)
         end
+        OpenFoodNetwork::TagRuleApplicator.new(
```

Comment:
there's a lot of recent history in these few spree lines...
I think/wonder:
- can the method shipping_categories be removed from this class all together?
- I think this code in this shipping_methods could be nonesense now: we are getting all ship methods in all ship categories so, naturally, all ship methods in any distributor will be included.
- this code is repeated in EnterpriseHelper#available_shipping_methods except for the "display_on_checkout" part (ship methods available on checkout only)

I think this code should be in a service DistributorShippingMethods and look simply like this:
```
      def shipping_methods
          order.distributor.shipping_methods.to_a
      end
```
plus the tagruleapplicator part, but we need to find a way to remove the duplication of the tagruleapplicator here and in the EnterpriseHelper